client-v2/src/components/FrontsEdit/CollectionComponents/Collection.tsx

```diff
@@ -54,37 +55,43 @@ const Collection = ({
   canPublish = true,
   publishCollection: publish,
   displayEditWarning,
-  isUneditable,
+  isCollectionUneditable,
   isOpen,
   onChangeOpenState
-}: CollectionProps) => (
-  <CollectionDisplay
-    frontId={frontId}
-    id={id}
-    browsingStage={browsingStage}
-    isUneditable={isUneditable}
-    isOpen={isOpen}
-    onChangeOpenState={() => onChangeOpenState(id, isOpen)}
-    headlineContent={
-      hasUnpublishedChanges &&
-      canPublish && (
-        <Button size="l" onClick={() => publish(id, frontId)}>
-          Launch
-        </Button>
-      )
-    }
-    metaContent={
-      alsoOn[id].fronts.length || displayEditWarning ? (
-        <CollectionNotification
-          displayEditWarning={displayEditWarning}
-          alsoOn={alsoOn[id]}
-        />
-      ) : null
-    }
-  >
-    {groups.map(group => children(group, isUneditable))}
-  </CollectionDisplay>
-);
+}: CollectionProps) => {
+  const isUneditable =
+    isCollectionUneditable || browsingStage !== collectionItemSets.draft;
+
+  return (
+    <CollectionDisplay
+      frontId={frontId}
+      id={id}
+      browsingStage={browsingStage}
+      isUneditable={isUneditable}
+      isLocked={isCollectionUneditable}
```

Comment:
The naming here is a bit confusing. The prop is `locked` but the value is `uneditable`, there is also `isUneditable` above which makes it slightly more confusing...

Not sure what a better naming looks like though...

Reply:
I need to run this by Akemi before merging to make sure this looks ok but if it does, re naming of variables, I'd be inclined to just call `isCollectionUneditable` `isCollectionLocked` instead and `isUneditable` could be called `isStageUneditable`. There are quite a few collection info types of things we display on collections in addition to this, coming from the fronts config so I'd be worried that just referring to `infoText` is too generic.